swan-lake/development-tutorials/source-code-dependencies/style-guide/statements.md

```diff
@@ -26,9 +26,9 @@ if true {
 
 
 ```ballerina
-if (true) {
+if valid {
     ...
-} else if (false) {
+} else if valid {
```

Comment:
This is the don't. So we have to have parentheses here.

Reply:
Fixed.